### PR TITLE
[18.0][MIG] base_m2m_custom_field: Migration to 18.0

### PR DESCRIPTION
**Pull request by trisdoan**

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

File: base_m2m_custom_field/readme/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+By setting create_table=False on the Many2manyCustom field, and using
+the relational table name, as \_name for the relational model, we're
+able to define a dedicated tree view for my.model.a.b.rel.
+
+``` xml
+<record id="my_model_a_b_rel_tree_view" model="ir.ui.view">
+    <field name="name">my.model.a.b.rel.tree.view</field>
+    <field name="model">my.model.a.b.rel</field>
+    <field name="arch" type="xml">
+        <tree editable="top">
```
Comment (joepsanders) [inline base_m2m_custom_field/readme/USAGE.md]: tree tag should be replaced with list

Comment (simahawk) [reply base_m2m_custom_field/readme/USAGE.md]: @trisdoan can you address this pls?

Comment (trisdoan) [reply base_m2m_custom_field/readme/USAGE.md]: sorry I missed this, will do

**Review by joepsanders — APPROVED** [review]

**Pull request merged**